releasing/run-goreleaser.sh

```diff
@@ -122,6 +122,12 @@ env:
 - GO111MODULE=on
 - GOWORK=off
 
+# Used to change the archive formats for specific GOOSs.
+# Most common use case is to archive as zip on Windows.
+format_overrides:
+  - goos: windows
+    format: zip
```

Comment:
Super nit comment: Please change indents like other lines if you can.


```suggestion
format_overrides:
- goos: windows
  format: zip
```